content/docs/setup/kubernetes/spec-requirements/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Requirements for Pods and Services
+title: Requirements for pods and services
```

Comment:
Actually, contrary to what Steve said, titles should have full capitalization.

Section headers (the ## stuff) is what uses initial-caps only.

I'll clarify that in the guidelines.

Reply:
@geeknoid my bad :)